src/patternfly/components/ModalBox/modal-box.scss

```diff
@@ -73,45 +84,68 @@
       margin-right: var(--pf-c-modal-box--c-button--sibling--MarginRight); // Create room for the close button
     }
   }
+}
 
-  > .pf-c-title {
-    @include pf-text-overflow;
+.pf-c-modal-box__header {
+  flex: 0 0 auto;
+  padding-top: var(--pf-c-modal-box__header--PaddingTop);
+  padding-right: var(--pf-c-modal-box__header--PaddingRight);
+  padding-left: var(--pf-c-modal-box__header--PaddingLeft);
 
-    flex: 0 0 auto;
+  &:last-child {
+    padding-bottom: var(--pf-c-modal-box__header--last-child--PaddingBottom);
+  }
 
-    + .pf-c-modal-box__description {
-      margin-top: var(--pf-c-modal-box__c-title--description--MarginTop);
-    }
+  + .pf-c-modal-box__body {
+    --pf-c-modal-box__body--PaddingTop: var(--pf-c-modal-box__header--body--PaddingTop);
+  }
 
```

Comment:
Looks like we lost `__title` + `__description` `padding-top: spacer--sm`. It's currently `spacer--xs`

Reply:
was changed in https://github.com/patternfly/patternfly/pull/2963 - compare to v4